### ISSUE TITLE
keep the style for specifying default values consistent

### ISSUE THREAD

**Issue by keewis** [Carreau/velin#21]
`numpydoc` [describes](https://numpydoc.readthedocs.io/en/latest/format.html#sections) multiple styles for adding default values to the type spec:
```
copy : bool, default True
copy : bool, default=True
copy : bool, default: True
```
(as far as I know, these are not the only ones out there, there's at least `bool (default True)`, too)

It would be nice to have velin make sure these are all in the same style. Not sure if autodetecting is possible / reliable, but a setting (as a template?) might work.

**Comment by Carreau** [comment]
I don't think that the Numpydoc does this extra parsing and just treat this as a string, but yes that would be nice. In particular it could also check/compare update WRT the type function type annotation.
